backrun/Cargo.toml

```diff
@@ -17,10 +17,10 @@ log = "0.4.14"
 prost-types = "0.8.0"
 rand = "0.8.5"
 searcher-service-client = { path = "../searcher_service_client" }
-solana-client = "=1.11.2"
-solana-metrics = "=1.11.2"
-solana-sdk = "=1.11.2"
-solana-transaction-status = "=1.11.2"
+solana-client = "=1.14.13"
+solana-metrics = "=1.14.13"
+solana-sdk = "=1.14.13"
```

Comment:
do we need jito-solana here to get bundle result errors?

Reply:
nvm, the message is parsed in the block engine (searcher_service) and sent here as string